.github/workflows/test.yml

```diff
@@ -10,27 +10,6 @@ on:
       - main
   workflow_call:
 jobs:
-  check-materialization:
```

Comment:
I don't understand why this is removed. What does this have to do with `eventuo11y`?

Reply:
Because apparently materialization does not actually speed up flake eval, which is the only reason we were doing it. So we're removing it so we have one fewer manual thing to manage.